_component_design/popovers.md

```diff
@@ -143,17 +147,18 @@ See [inputs and controls]({{site.baseurl}}/components/inputs-and-controls.html)
 
 ### Pointer size and placement
```

Comment:
Redline change: the spacing specs looks out of place on this image. it's not a big deal but we should probably use a span redline for that rather than a spacing block.